src/general/cpuInfo.go

```diff
@@ -47,29 +47,51 @@ func NewCPULoad() *CPULoad {
 	return &CPULoad{}
 }
 
+// ReadCPULoad reads /proc/stat and returns the average load
+func ReadCPULoad() ([10]float64, error) {
+	data, error := os.ReadFile("/proc/stat")
```

Comment:
Could this be made into a slice instead of an array? If not, maybe as a struct as it would make the fields more readable.

Reply:
Sure I'll update it tomorrow. Never really used Go before so I need to figure out what a slice is xD